exchange/exchange_test.go

```diff
@@ -1800,6 +1800,14 @@ func (big *mockBidIDGenerator) New() (string, error) {
 
 }
 
+type MockRandomBooleanGenerator struct {
```

Comment:
Nitpick: Consider using the [term 'Fake' instead of 'Mock here](https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing#:~:text=Fake%3A%20a%20class%20that%20implements,fixed%20data%20and%20no%20logic.&text=Mock%3A%20a%20class%20that%20implements,have%20been%20called%2Fnot%20called.).

Reply:
Renamed